property/index.js

```diff
@@ -166,3 +161,62 @@ module.exports = yeoman.generators.Base.extend({
 
   saveProject: actions.saveProject
 });
+
+function coerceDefault(def, value) {
+  var isSupported = typeChoices.indexOf(def.type) !== -1;
+  debug('property type "%s" >> supported: %s', def.type, isSupported);
+
+  if (isSupported) {
+    switch ((def.type || '').toLowerCase()) {
+      case 'string':
+        def.default = value;
+        break;
+      case 'number':
+        def.default = Number(value);
+        break;
+      case 'boolean':
+        if (['true', '1', 't'].indexOf(value) !== -1 ){
+          def.default = true;
+        } else {
+          def.default = false;
+        }
+        break;
+      case 'object':
+        def.default = JSON.parse(value);
+        break;
+      case 'array':
+        def.default = value.replace(/[\s,]+/g, ',').split(',');
+        break;
+      case 'date', 'datetime':
+        if (value.toLowerCase() === 'now'){
+          def.defaultFn = 'now';
+        } else {
+          def.default = new Date(value);
```

Comment:
Is `value` always a string? I know this came up in strong-remoting, but if `value` is of JS-type number, it will give a different date than JS-type string.

I'm not sure that we handled this before but we should deal with it now. Or tell the user that anything from the command line will be parsed as a string so they should expect the correct output accordingly.